src/pages/Auth/Login.jsx

```diff
@@ -1,3 +1,108 @@
+import "./auth.css";
+import { Link, useNavigate } from "react-router-dom";
+import { FaEyeSlash, FaEye } from "react-icons/fa";
+import { useState } from "react";
+import { Button } from "../../components";
+import axios from "axios";
+import { useAuth } from "../../contexts";
 export const Login = () => {
-  return <div>Login Page</div>;
+  const initialData = {
+    email: "",
+    password: "",
+  };
+  const testCredentials = {
```

Comment:
use this for guest login, it will be helpful

Reply:
I'm using that for guest Login